vita3k/kernel/src/thread/sync_primitives.cpp

```diff
@@ -583,7 +583,6 @@ int condvar_signal(KernelState &kernel, const char *export_name, SceUID thread_i
 
                 waiting_thread->something_to_do.notify_one();
             }
-
```

Comment:
:eyes:

Reply:
format script removed this line, should i add it back?

Reply:
I see. The main reason I highlighted it was because it was in a module not really related to the pr.

If you don't want to bother adding it back then it's okay. I don't think the line should've been there anyway.